FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -327,7 +329,7 @@ - (bool)noRunningConnection {
 }
 
 - (bool)canMakeConnection {
-  return [self noRunningConnection] && [self->_listeners count] > 0;
+  return [self noRunningConnection] && [self->_listeners count] > 0 && !gIsInBackground;
```

Comment:
reading needs to be guarded by a serial queue as well

Reply:
Moved all of these vars to be wrapped by serial queue.